app/helpers/requests_helper.rb

```diff
@@ -4,4 +4,12 @@ module RequestsHelper
   def replace_whitespaces(name)
     name.parameterize(preserve_case: true)
   end
+
+  def mb_to_gb(megabytes)
```

Comment:
Are we talking about GigaBytes and MegaBytes or GibiBytes and MibiBytes? if we are talking about Gibi, we would have to work with 1024 instead of 1000

Reply:
We are really talking about Megabyte and Gigabyte